src/lms-harmonizer/edfi_lms_harmonizer/harmonizer.py

```diff
@@ -18,27 +18,18 @@
 
 
 def _generate_call_to_stored_procedure(
-    engine: str, procedure: str, parameters: dict[str, str] = dict()
+    engine: str, procedure: str, namespace: str = None, source_system: str = None
```

Comment:
Once I did the coding for parameters, I realized they were all the same and it was overkill. Good simplification here.